### DISCUSSION TITLE
Show All Fields disappears after use

### DISCUSSION THREAD

**Discussion by alekmlynek** [laravel/nova-issues#4190] [Ideas & Feature Requests]
- Laravel Version: 9.11.0
- Nova Version: 4.4.2
- PHP Version: 8+

### Description:
When using the ->limit(x) function on Panels, Nova will add a "show all fields" link on the Resource detail page that will expand the panel when clicked and show all fields past limit.

### Problem:
Once the "Show all fields" is clicked, it disappears and there is no way to get it back except for refreshing the page.

### Detailed steps to reproduce the issue on a fresh Nova installation:
1. Add a ->limit(x) to any Nova panel
2. Click "Show All fields"
3. It disappears after use

**Comment by crynobone** [comment]
That's the expected behavior.

**Comment by alekmlynek** [comment]
Then I would say that is an odd and inconstant choice.

All show/hide content, relationships, side-menus and everywhere else has toggle enabled. This is the only area in the whole system that is not.

The issue was brought to me by a client, they didn't think it was working right, and I agreed with them. I'm not sure what else to say, other then it looks like a bug to users.
